config/webpack/production.js

```diff
@@ -1,3 +1,12 @@
 const environment = require('./environment')
+const { merge } = require('webpack-merge')
+
+const sassLoader = environment.loaders.get('sass');
+
+sassLoader.use.map(function(loader) {
```

Comment:
To avoid inevitable Prettier conflicts:

```suggestion
sassLoader.use.map((loader) => {
```

Or at least in Prettier format of function statement

```suggestion
sassLoader.use.map(function (loader) {
```